scripts/clang-format.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+: '
+
+Runs clang-format on the code in src/
+
+Return `1` if there are files to be formatted, and automatically formats them.
+
+Returns `0` if everything looks properly formatted.
+
+'
+
+PATH_TO_FORMAT_SCRIPT="$(pwd)/mason_packages/.link/bin/clang-format"
```

Comment:
Why do these scripts depend on mason? mason should put clang-format into PATH and these scripts should not all have to know about mason.